modules/longhorn/variables.tf

```diff
@@ -0,0 +1,29 @@
+
+variable "ssh_config" {
+  description = "Connection details to apply configuration"
+  type = object({
+    host        = string
+    user        = optional(string)
+    private_key = string
+  })
+}
+
+variable "addon_context" {
+  description = "Input context for the addon"
+  type = object({
+    equinix_project        = string
+    equinix_metro          = string
+    kubeconfig_remote_path = string
+  })
+}
+
+variable "addon_config" {
+  description = "Add-on configuration for Longhorn"
+  type        = any
+  //TODO should we define all options here? 
+  # type = object({
```

Comment:
nit:  this commented example isn't relevant for the longhorn add-on (although it looks like this variable could be removed entirely?)

Reply:
Agreed, nuked it.